services/openvpn/service/factory.go

```diff
@@ -121,9 +124,11 @@ func newServerFactory(nodeOptions node.Options, sessionValidator *openvpn_sessio
 			}
 		}
 
+		protectedNetworks := strings.Split(node_config.GetString(node_config.FlagFirewallProtectedNetworks), ",")
 		return openvpn.CreateNewProcess(
 			nodeOptions.Openvpn.BinaryPath(),
 			config.GenericConfig,
+			filter.NewMiddleware(nil, protectedNetworks),
```

Comment:
`nil`?

Reply:
It's a list of addresses that should be excluded from the block list. We are going to use it in the future when we will need to allow some hosts, like DNS servers. 

It will be looking like this:

`filter.NewMiddleware([]string{"10.10.10.1/32"}, []string{"10.0.0.0/8"})`

Reply:
anyway would be to pass array instead of nil-able array

Reply:
nil slice is actually idiomatic in go